src/Sensors/ExceptionSensor.php

```diff
@@ -102,9 +103,22 @@ private function wasManuallyReported(Throwable $e): bool
      */
     private function serializeTrace(Throwable $e): string
     {
-        $trace = [];
+        $trace = [
+            // Insert the exception location as the first frame.
+            // This matches the behavior of Symfony's exception renderer.
+            [
+                'file' => $this->location->normalizeFile($e->getFile()).':'.$e->getLine(),
+                'source' => '',
+            ],
+        ];
+
+        foreach ($e->getTrace() as $i => $frame) {
+            if (($i === 0 || $i === 1) && ($frame['class'] ?? '') === HandleExceptions::class) {
+                // Skip internal frames when a PHP error has been converted to an ErrorException
+                // This matches the behavior of Laravel's exception renderer.
+                continue;
+            }
```

Comment:
This is similar to Laravel's approach:

https://github.com/laravel/framework/blob/ac498ff3adb6c434cd699f9085bb8bec66754316/src/Illuminate/Foundation/Exceptions/Renderer/Exception.php#L118-L121